### PR TITLE
[UX] Empty student plans

### PR DESCRIPTION
**Pull request by maciejhein**
Screenshot 2019-09-23 at 17 31 23

Screenshot 2019-09-23 at 11 52 10

### CODE REVIEW

File: src/locale/en.ts [hunk 1]
```diff
@@ -69,7 +69,9 @@ const translations = {
   },
   planList: {
     viewTitle: 'All Plans',
-    createPlan: 'Add a Plan',
+    createPlan: 'DODAJ NOWY PLAN',
+    copyPlan: 'SKOPIUJ ISTNIEJĄCY PLAN',
```
Comment (bklukaczewski) [inline src/locale/en.ts]: I think it would be better to use style properties for uppercase `textTransform: 'uppercase'` instead of translation files

Comment (bklukaczewski) [reply src/locale/en.ts]: And it's english translation file btw

File: src/screens/emptyStudentPlans/EmptyStudentPlansScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <CopyPlanButton />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
```
Comment (bklukaczewski) [inline src/screens/emptyStudentPlans/EmptyStudentPlansScreen.tsx]: It's default value in yoga layout ;)

Comment (maciejhein) [reply src/screens/emptyStudentPlans/EmptyStudentPlansScreen.tsx]: fixed

File: src/locale/en.ts (hunk 1 above)
Comment (maciejhein) [reply src/locale/en.ts]: fixed

**Pull request merged**